tests/pom.xml

```diff
@@ -16,7 +16,7 @@
     <!--
     These properties are read in by arquillian.xml
     -->
-    <browser>phantomjs</browser> <!-- PhantomJS will be our default browser if no profile is specified-->
+    <browser>htmlUnit</browser> <!-- htmlUnit will be our default browser if no profile is specified-->
```

Comment:
@starksm64 Any reason to stick with htmlunit instead of PhantomJS?  No support for JavaScript, Rhino engine not implemented in any major browsers, etc..